Regression/WarpX-tests.ini

```diff
@@ -798,6 +798,24 @@ particleTypes = electrons
 analysisRoutine = Examples/analysis_default_regression.py
 tolerance = 1.e-14
 
+[LaserAccelerationRestart]
```

Comment:
By default, does this test run a post-processing check, where the results of the restarted simulation are compared with the results of the original simulation?

If not, I would suggest to instead replace the beam driver of our existing restart test by a laser driver. I can help with this.

Reply:
I thought it did, but now I see that, when we through the CI, we set `doComparison=0` for all tests, which disables comparing against the simulation. So I guess we need to do this via an analysis script, as you suggest.

Reply:
Sounds good. I will update the PR (along with some other minor changes), hopefully within the next 2-3 hours. 
For now, I will set the PR to `[WIP]`.